config.test.yaml

```diff
@@ -39,6 +39,7 @@ templates:
       - mediawiki/v1/content
       - mediawiki_v1_graphoid
       - mediawiki/v1/mobileapps
+      - analytics_v1_pageviews
```

Comment:
@milimetric We've settled on a directory-based structure of configs, so this should go to `analytics/v1/pageviews.yaml`